cmd/crawler/main.go

```diff
@@ -31,7 +32,26 @@ func main() {
 		return
 	}
 
-	client, _ := db.NewDGraph()
+	// Get filename from args
+	filename := "../../config.json"
+	if len(args) == 4 {
+		filename = args[3]
+	}
+
+	// Get config from file
+	file, err := os.Open(filename)
+	if err != nil {
+		log.Fatal("Couldn't open config file")
+	}
+
+	decoder := json.NewDecoder(file)
+	var config db.Config
+	err = decoder.Decode(&config)
+	if err != nil {
+		log.Fatal("Couldn't parse config file")
+	}
+
+	client, _ := db.NewDGraph(config)
```

Comment:
I know it's not directly related to this PR, but can we catch the error that can be returned by `NewDGraph` here? 😄